src/Libraries/Microsoft.Extensions.Http.Resilience/Hedging/Internals/Validators/HttpStandardHedgingResilienceOptionsValidator.cs

```diff
@@ -2,11 +2,10 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 
 using Microsoft.Extensions.Options;
-using Microsoft.Extensions.Options.Validation;
 
 namespace Microsoft.Extensions.Http.Resilience.Internal.Validators;
 
-[OptionsValidator]
-internal sealed partial class HttpStandardHedgingResilienceOptionsValidator : IValidateOptions<HttpStandardHedgingResilienceOptions>
+internal sealed class HttpStandardHedgingResilienceOptionsValidator : IValidateOptions<HttpStandardHedgingResilienceOptions>
 {
+    public ValidateOptionsResult Validate(string? name, HttpStandardHedgingResilienceOptions options) => ValidateOptionsResult.Success;
```

Comment:
Why this change? The prior pattern would achieve the same effect.

Reply:
I'll restore the validation when alpha.4 is ready.

Reply:
Fixed.